package.json

```diff
@@ -3,15 +3,20 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "engines": {
+    "node": ">=16.20.2",
+    "pnpm": ">=8.14"
+  },
   "scripts": {
-    "lint-project": "cd project && npm run lint",
-    "ci-project": "cd project && npm ci && npm run build",
-    "pack": "cd project && npm pack",
-    "unit-test": "cd project && npm run test-ci",
-    "ci-integration-tests": "cd integration-tests && npm ci && npm run build",
-    "integration-test": "cd integration-tests && npm run test-ci",
-    "lint-test": "cd integration-tests && npm run lint",
-    "release": "npm run copy-readme && cd project && npm run release",
+    "preinstall": "npx only-allow pnpm",
+    "lint-project": "pnpm -C project lint",
+    "ci-project": "pnpm -C project build",
+    "package": "pnpm -C project package",
+    "unit-test": "pnpm -C project test-ci",
+    "ci-integration-tests": "pnpm package && pnpm -C integration-tests add -D 'file://../project/anev-ts-mountebank-1.8.2.tgz' && pnpm -C integration-tests build",
+    "integration-test": "pnpm package && pnpm -C integration-tests add -D 'file://../project/anev-ts-mountebank-1.8.2.tgz' && pnpm -C integration-tests test-ci",
```

Comment:
This does not work locally for me anymore, unless I start MB manually.
How do you run them locally?

What do you think about this:
1. rename `ci-integration-tests` => `build:integration-tests` 
2. `integration-test` => rename to `integration-tests-ci`
3. new script for running local similar to the 'integration-test-ci` but that does not start MB.

Reply:
Applied the alias name changes and new local `integration-test-local` w/o starting `mb`. See https://github.com/AngelaE/ts-mountebank/pull/27/commits/ebe4601373c1568996fbdf41098e181960d0396a